Formula/php.rb

```diff
@@ -0,0 +1,274 @@
+      --enable-sysvmsg
+      --enable-sysvsem
+      --enable-sysvshm
+      --enable-wddx
+      --enable-zip
+      --libexecdir=#{libexec}
+      --with-apxs2=#{Formula["httpd"].opt_bin}/apxs
+      --with-bz2=/usr
+      --with-enchant=#{Formula["enchant"].opt_prefix}
+      --with-freetype-dir=#{Formula["freetype"].opt_prefix}
+      --with-gmp=#{Formula["gmp"].opt_prefix}
+      --with-gd
+      --with-gettext=#{Formula["gettext"].opt_prefix}
+      --with-fpm-user=_www
+      --with-fpm-group=_www
+      --with-iconv-dir=/usr
+      --with-icu-dir=#{Formula["icu4c"].opt_prefix}
+      --with-jpeg-dir=#{Formula["jpeg"].opt_prefix}
+      --with-kerberos=/usr
+      --with-ldap=shared
+      --with-ldap-sasl=/usr
+      --with-libxml-dir=/usr
+      --with-imap=shared,#{Formula["imap-uw"].opt_prefix}
+      --with-imap-ssl=#{Formula["openssl"].opt_prefix}
+      --with-mhash
+      --with-mysql-sock=/tmp/mysql.sock
+      --with-mysqli=mysqlnd
+      --with-pdo-mysql=mysqlnd
+      --with-pdo-odbc=unixODBC,#{Formula["unixodbc"].opt_prefix}
+      --with-ndbm=/usr
+      --with-openssl=#{Formula["openssl"].opt_prefix}
+      --with-pdo-dblib=#{Formula["freetds"].opt_prefix}
+      --with-pdo-pgsql=#{Formula["libpq"].opt_prefix}
+      --with-pgsql=#{Formula["libpq"].opt_prefix}
+      --with-png-dir=#{Formula["libpng"].opt_prefix}
+      --with-pspell=#{Formula["aspell"].opt_prefix}
+      --with-snmp
+      --with-tidy=shared,#{Formula["tidy-html5"].opt_prefix}
+      --with-unixODBC=#{Formula["unixodbc"].opt_prefix}
+      --with-webp-dir=#{Formula["webp"].opt_prefix}
+      --with-pic
+      --with-xmlrpc
+      --with-zlib=/usr
+      --with-libedit
+      --with-xsl=/usr
+    ]
+
+    if MacOS.version < :lion
+      args << "--with-curl=#{Formula["curl"].opt_prefix}"
+    else
+      args << "--with-curl"
+    end
+
+    args << "--enable-maintainer-zts" if build.with? "thread-safety"
+
+    if build.devel?
+      args += %W[
+        --with-password-argon2=#{Formula["argon2"].opt_prefix}
+        --with-sodium=#{Formula["libsodium"].opt_prefix}
+      ]
+    else
+      args << "--with-mcrypt=shared,#{Formula["mcrypt"].opt_prefix}"
+    end
+
+    system "./buildconf", "--force"
+    system "./configure", *args
+
+    inreplace "Makefile" do |s|
+      s.gsub! /^INSTALL_IT = \$\(mkinstalldirs\) '([^']+)' (.+) LIBEXECDIR=([^\s]+) (.+) -a (.+)$/,
+        "INSTALL_IT = $(mkinstalldirs) '#{libexec}/apache2' \\2 LIBEXECDIR='#{libexec}/apache2' \\4 \\5"
+
+      %w[EXTRA_LDFLAGS EXTRA_LDFLAGS_PROGRAM].each do |mk_var|
+        system_libs = []
+        other_flags = []
+        s.get_make_var(mk_var).split.each do |f|
+          if f[%r{^-L\/(?:Applications|usr\/lib)\/}]
+            system_libs << f
+            next
+          end
+          other_flags << f
+        end
+        s.change_make_var! mk_var, other_flags.concat(system_libs).join(" ")
+      end
+    end
+
+    ENV.prepend "LDFLAGS", "-L#{Formula["tidy-html5"].opt_lib}"
```

Comment:
You already set `--with-tidy=shared,#{Formula["tidy-html5"].opt_prefix}`. Is this a bug?

Reply:
Yes, it's a bug. The Makefile's ordering of library flags for this shared module will pick up the system's `libtidy` (wrong version) without this flag.